src/Results/Results.tsx

```diff
@@ -9,19 +9,30 @@ const Results: React.FC<ResultsProps> = props => {
 
 	const noResults = 'I\'m sorry, there are no results. Please try again!'
 
-	const resultsList = props.searchResults.map(result => {
+	const americanResults = props.searchResults.filter((result) => {
+		return result.country === "United States";
+	}); 
+
+	const noDupCardInfo = (label: string) => {
```

Comment:
Thanks for adding this! I think it makes the results page look much better. I agree, there was a lot of duplicated information.